spec/services/audit/catalog_to_archive_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe Audit::CatalogToArchive do
-  let(:zmv) { create(:zipped_moab_version) }
+  let(:zmv) { create(:zipped_moab_version, preserved_object: create(:preserved_object_fixture, druid: 'bz514sm9647')) }
```

Comment:
Without this the zipped moab version's preserved object does not have a complete moab.